### DISCUSSION TITLE
Data Seeding ?

### DISCUSSION THREAD

**Discussion by utamori** [ariga/atlas#1263] [Q&A]
How can we seed testdata after schema migration for testing?

**Comment by rotemtam** [comment]
Hey @utamori 

Great seeing you here !

Are you using declarative migrations or versioned migrations?

**Reply by utamori** [reply]
declarative migrations or Authoring!

**Comment by cemo** [comment]
This is a topic we are also interested a lot. 

We use declarative migrations for schema management. However for data management we use versioned migrations.
